lld/ELF/InputFiles.cpp

```diff
@@ -1797,18 +1799,18 @@ void BitcodeFile::parse() {
 void BitcodeFile::parseLazy() {
   numSymbols = obj->symbols().size();
   symbols = std::make_unique<Symbol *[]>(numSymbols);
-  for (auto [i, irSym] : llvm::enumerate(obj->symbols()))
+  for (auto [i, irSym] : llvm::enumerate(obj->symbols())) {
+    // Keep copies of per-module undefined symbols for LTO::GlobalResolutions
+    // usage.
+    irSym.Name = unique_saver().save(irSym.getName());
     if (!irSym.isUndefined()) {
       // Symbols can be duplicated in bitcode files because of '#include' and
       // linkonce_odr. Use unique_saver to save symbol names for de-duplication.
```

Comment:
The unique_saver is now moved out, so should this comment be moved up too and merged with the "Keep copies of per-module undefined symbols" in some way?

Reply:
done.